build/templates/backend/scripts/polygon_heimdall.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+{{define "main" -}}
+
+set -e
+
+INSTALL_DIR={{.Env.BackendInstallPath}}/{{.Coin.Alias}}
+DATA_DIR={{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend
+
+HEIMDALL_BIN=$INSTALL_DIR/heimdalld
+HOME_DIR=$DATA_DIR/heimdalld
+CONFIG_DIR=$HOME_DIR/config
+
+if [[ ! -d "$CONFIG_DIR" ]]; then
```

Comment:
When running `heimdall`, I get error report
```
/opt/coins/nodes/polygon_heimdall_archive/polygon_heimdall_archive_exec.sh: 10: [[: not found
```
In bsc you use this form:
```
if [ ! -d "$CHAINDATA_DIR" ]; then
```

Reply:
https://github.com/trezor/blockbook/pull/872/commits/9a5f195eb356cacfd01df684e73f6a50f0741776